config/bootstrap.php

```diff
@@ -155,7 +155,7 @@
     }
 
     $httpHost = env('HTTP_HOST');
-    if (isset($httpHost)) {
+    if ($httpHost !== null) {
```

Comment:
I wonder if this should even be 
```suggestion
    if ($httpHost) {
```
As empty string would also be weird here